src/HTMLGenerator/testSuite/sequencer.py

```diff
@@ -36,8 +70,8 @@ def main():
                     try:
                         with open(json_tests_directory/json_test) as json_test_file:
                             test_cases_json = json.load(json_test_file)
-                            converter_error = convert(top_level_output_directory, test_assets, test_cases_json, Path.cwd().parent)
-
+                            
+                            converter_error = convert_test_suite(top_level_output_directory, test_assets, test_cases_json, Path.cwd().parent)
```

Comment:
I would recommend renaming `convert_test_suite()` to `convert_test_case()`. The code does not convert the entire test suite in one go, it converts every test case in the test suite.

Reply:
Done